src/pyhf/version.py

```diff
@@ -1,5 +0,0 @@
-"""Define pyhf version information."""
```

Comment:
I guess we should also add `src/pyhf/version.py` to `.gitignore` so that once it is generated we know to ignore it.